index.js

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable max-classes-per-file */
+
```

Comment:
- [ ] As much as possible, kindly avoid disabling  inline linter checks. This is because they represent the coding style adopted by your team  and also most of the time suggest best coding practices.